docker-compose.yml

```diff
@@ -827,6 +827,22 @@ services:
         max-size: "${DEFAULT_LOG_SIZE}"
         max-file: "${HIGH_PRIO_LOG_FILES}"
 
+  #
+  # FAF Solr standalone server
+  #
+  faf-solr:
+    # version: '3'
```

Comment:
This isn't part of service definitions, it only specifies the docker-compose file version itself. Remove it here.